src/jit_runner.cpp

```
@@ -0,0 +1,374 @@
+                errs() << "Failed to load native library '" << libName << "': ";
+                logAllUnhandledErrors(libGen.takeError(), errs());
+                ry::emitTraceEvent("runtime.error", "jit", &sourceLoc,
+                                   {ry::TraceField("detail", "Failed to load library: " + libName)});
+                return 1;
+            }
+            mainJD.addGenerator(std::move(*libGen));
+        }
+    }
+
+    // Register test runtime symbols if in test mode
+    if (test_mode) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)